scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -271,6 +271,35 @@ def scc_order(self):
         except Exception as e:
             raise EDTError(e)
 
+    def _link_properties(self, link_node, node):
```

Comment:
Variable names here are not clear. Can you please pick names which make it clear which is the source node and which is the dest node in the dependency digraph or otherwise document this more clearly?